controller/delete_broken_metadata_test.go

```diff
@@ -103,7 +103,7 @@ func TestDeleteBrokenMetadata(t *testing.T) {
 
 			ctrl := controller.New("http://asd", "/v1", "asdasd", metadataStorage, contentStorage, nil, logger)
 
-			router, _ := ctrl.SetupRouter(nil, "/v1", ginLogger(logger))
+			router, _ := ctrl.SetupRouter(nil, "/v1", []string{}, false, ginLogger(logger))
```

Comment:
I think you need to set this to `[]string{"*"}` which was the previous setting based on your changes.